os/arch/arm/src/common/up_checkstack.c

```diff
@@ -72,6 +72,25 @@
 
 #ifdef CONFIG_STACK_COLORATION
 
+#ifndef CONFIG_STACK_ALIGNMENT
+
+/* The symbol  __ARM_EABI__ is defined by GCC if EABI is being used.  If you
+ * are not using GCC, make sure that CONFIG_STACK_ALIGNMENT is set correctly!
+ */
+
+#ifdef __ARM_EABI__
+#define CONFIG_STACK_ALIGNMENT 8
+#else
+#define CONFIG_STACK_ALIGNMENT 4
+#endif
+#endif
+
+/* Stack alignment macros */
+
+#define STACK_ALIGN_MASK    (CONFIG_STACK_ALIGNMENT-1)
```

Comment:
(CONFIG_XX - 1)
please add spaces around `-`.

Reply:
Thank you, Fixed!~